contracts/base/Loans.sol

```diff
@@ -361,4 +361,15 @@ contract Loans is LoansInterface, SignerRole {
             liquidated: false
         });
     }
+
+    function isCollateralSentEnough(
+        uint256 msgValue,
+        uint256 amountToBorrow,
+        uint256 collateralRatio
+    ) internal view returns (bool) {
+        uint256 oneLendingTokenPriceWeis = uint256(priceOracle.getLatestAnswer());
```

Comment:
I believe there is a disconnect between the values in this contract. Some are looking at USDC in whole tokens, and some are looking at USDC in units of tokens.

`oneLendingTokenPriceWeis` does not have the token decimals incorporated yet. From the screenshot you shared yesterday of the oracle response, at a price of 200 USDC for 1 ETH, the contract would return the following:
1 USDC = 5000000000000000 WEI = 0.005 ETH.
However this is 1 whole USDC, not 1 unit of USDC.
So we have `oneLendingTokenPriceWeis = 5000000000000000`
Then if 2 ETH was sent, `collateralSent = (2 ETH / 0.005 ETH) = 400` => i.e. 400 USDC worth

However, `amountToBorrow` is used in the Loans contract as follows:
`lendingPool.createLoan(amountBorrow, msg.sender);`
which in turn performs:
`tokenTransfer(borrower, amount);`
This means that `amountToBorrow` IS incorporating the units of the token. If you want to have 400 whole USDC, you have to call `lendingPool.createLoan(400*(10**6), msg.sender)`.

Reply:
Now, it considers the lending token decimals to verify whether collateral sent is enough or not.